packages/itwinui-css/src/avatar/avatar.scss

```diff
@@ -3,59 +3,39 @@
 @use '../mixins';
 @use './mixins' as *;
 
+$avatar-text-color: hsl(0 0% 0% / 0.6); // same as badge, meets AA contrast against soft backgrounds
+
 .iui-avatar {
   @include mixins.iui-reset;
   display: inline-flex;
   align-items: center;
   justify-content: center;
   position: relative;
-  margin: 0;
-  user-select: none;
   inline-size: var(--_iui-avatar-size);
   block-size: var(--_iui-avatar-size);
-  background-color: hsl(72, 51%, 56%);
+  background-color: var(--iui-color-background-seagreen);
   border-radius: 50%;
-
-  > .iui-stroke {
-    display: block;
+  font-weight: var(--iui-font-weight-semibold);
+  text-align: center;
+  text-transform: uppercase;
+  color: #{$avatar-text-color};
+  line-height: var(--_iui-avatar-size);
+  font-size: var(--_iui-avatar-font-size);
+
```

Comment:
Currently the border gets covered up when an image is displayed.
![Screenshot 2023-07-28 at 3 24 10 PM](https://github.com/iTwin/iTwinUI/assets/849817/b849d5ef-5659-444f-95d4-e2042922a637)

This can be fixed if we add:
```scss
.iui-avatar {
  isolation: isolate;

  &::before {
    z-index: 1;
  }

  &[data-iui-status] {
    &::after {
      z-index: 2;
    }
  }
}
```

![Screenshot 2023-07-28 at 3 27 04 PM](https://github.com/iTwin/iTwinUI/assets/849817/6cb0aff6-f0a9-4d26-8b6a-514e6bd9fb9c)

Reply:
fixed